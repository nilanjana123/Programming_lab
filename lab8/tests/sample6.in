#sdmt

10

2 3
100,200,300,
400,500,600,

5

3 2
1,2,
3,4,
5,6,

500
700
