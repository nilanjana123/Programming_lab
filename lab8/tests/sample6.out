# cmd options :sdmt
# After subtraction.
2 3
90,190,290,
390,490,590,

# After division.
2 3
18,38,58,
78,98,118,

# After multiplication.
2 2
422,536,
962,1256,

# After transpose.
2 2
422,962,
536,1256,
